src/components/interactive-map/interactive-map.tsx

```diff
@@ -143,18 +148,18 @@ const InteractiveMap: React.FC<InteractiveMapProps> = (props) => {
                 <circle
                   cx={province.center.x}
                   cy={province.center.y}
-                  r={5}
-                  fill="none"
+                  r={7}
+                  fill="white"
                   stroke="black"
-                  strokeWidth={1}
+                  strokeWidth={2}
                 />
                 <circle
                   cx={province.center.x}
                   cy={province.center.y}
-                  r={3}
-                  fill="none"
+                  r={4}
+                  fill="white"
                   stroke="black"
-                  strokeWidth={1}
+                  strokeWidth={2}
```

Comment:
Can we extract all of these values into named constants:
- `SUPPLY_CENTER_SIZE_INNER`
- `SUPPLY_CENTER_SIZE_OUTER` 
- `SUPPLY_CENTER_FILL`
- `SUPPLY_CENTER_STROKE`
- `SUPPLY_CENTER_STROKE_WIDTH`